feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/domain/NetworkManagementChainInteractor.kt

```diff
@@ -77,11 +77,23 @@ class RealNetworkManagementChainInteractor(
 
     override suspend fun toggleAutoBalance(chainId: String) {
         val chain = chainRegistry.getChain(chainId)
-        chainRegistry.setAutoBalanceEnabled(chainId, chain.autoBalanceDisabled)
```

Comment:
`chain.autoBalanceDisabled` - this was extremely unclear, we should not use such sort of negation